src/test/java/TestMultithreading/ListFill.java

```diff
@@ -0,0 +1,31 @@
+package TestMultithreading;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public interface ListFill {
```

Comment:
```suggestion
public class ListFill {
```
there is no point in interface with default methods only